Jenkinsfile

```diff
@@ -27,7 +27,7 @@ if (needSplittingFromWorkspace) {
 for (int i = 0; i < splits.size(); i++) {
   int index = i
   for (int j in [11]) {
-    for (String v in ['lts', 'latest']) {
+    for (String v in ['latest']) {
```

Comment:
```suggestion
    for (String v in ['lts', 'latest']) {
```

Reply:
if the change is going to be merged upstream the ATH needs to be compatible with the old way of doing things and the new way.  having a pre-tested change that works both with and without the PR will minimze the time to adapt if the upstream is merged (there would not be any possibility of regressions here).